packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -260,15 +261,23 @@ async def download_file_from_link(
 
 
 async def _abort_upload(
-    session: ClientSession, upload_links: FileUploadSchema, *, reraise_exceptions: bool
+    session: ClientSession,
+    upload_links: Optional[FileUploadSchema],
+    *,
+    reraise_exceptions: bool,
 ) -> None:
+    if not upload_links:

```

Comment:
can you explain this use-case please?

Reply:
I think below comments explain this. Removed form here